index.html

```diff
@@ -0,0 +1,371 @@
+							<hr class="bar"> </hr>
+						</div>
+					
+					<!-- program of the day -->
+					<div class="program-content container">
+						<div
+							class="d-flex justify-content-between align-content-center shadow p-2 mb-2 bgColor3"
+							id="program-info">
+							<div class="program-img">
+								<img
+									class="text-center"
+									src="./asset/images/program_lecture.png"
+									alt="lecture" />
+							</div>
+
+							<h3
+								class="text-danger text-center display-5 px-2"
+								id="program-task">
+								LECTURE
+							</h3>
+
+							<p class="text-light">
+								Listen to the speakers from various experience about the of
+								sharing of experience and opening.
+							</p>
+						</div>
+
+						<div
+							class="d-flex justify-content-between align-content-center shadow p-2 mb-2 bgColor3"
+							id="program-info">
+							<div class="program-img">
+								<img
+									class="text-center"
+									src="./asset/images/program-exhibition.png"
+									alt="lecture" />
+							</div>
+
+							<h3
+								class="text-danger text-center px-2 display-5"
+								id="program-task">
+								Exhibition
+							</h3>
+
+							<div>
+								<p class="text-light">
+									Appreciate various creations applying CC license of artists,
+									organized from Art Center Nabi
+								</p>
+							</div>
+						</div>
+
+						<div
+							class="d-flex justify-content-between align-content-center shadow p-2 mb-2 bgColor3"
+							id="program-info">
+							<div class="program-img">
+								<img
+									class="text-center"
+									src="./asset/images/program_forum.png"
+									alt="lecture" />
+							</div>
+
+							<h3
+								class="text-danger text-center px-2 display-5"
+								id="program-task">
+								Forum
+							</h3>
+
+							<p class="text-light">
+								Have the time to share your thoughts and opinions with experts
+								for each topic.
+							</p>
+						</div>
+
+						<div
+							class="d-flex justify-content-between align-content-center shadow p-2 mb-2 bgColor3"
+							id="program-info">
+							<div class="program-img">
+								<img
+									class="text-center"
+									src="./asset/images/program_workshop.png"
+									alt="lecture" />
+							</div>
+
+							<h3
+								class="text-danger text-center px-2 display-5"
+								id="program-task">
+								Work shop
+							</h3>
+
+							<p class="text-light">
+								Try creating your own work using open source license rather than
+								just watching at it.
+							</p>
+						</div>
+
+						<div
+							class="d-flex justify-content-between align-content-center shadow p-2 mb-2 bgColor3"
+							id="program-info">
+							<div class="program-img">
+								<img
+									class="text-center"
+									src="./asset/images/program_opportunities.png"
+									alt="lecture" />
+							</div>
+
+							<h3
+								class="text-danger text-center px-2 display-5"
+								id="program-task">
+								Ignite
```

Comment:
[REQUIRED] Kindly reduce the size of this text so that it can match the template. It is a requirement for your design to match the template in terms of layout, spacing, colour and typography.